adele-event/src/main/java/com/microservicesteam/adele/event/domain/Coordinates.java

```diff
@@ -1,19 +1,28 @@
 package com.microservicesteam.adele.event.domain;
 
-import org.immutables.value.Value;
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
 
-@Value.Immutable
-public interface Coordinates {
-    
-    double latitude();
-
-    double longitude();
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Value;
+import lombok.experimental.FieldDefaults;
 
-    class Builder extends ImmutableCoordinates.Builder {
-    }
-
-    static Builder builder() {
-        return new Builder();
-    }
+@Value
+@Getter(AccessLevel.NONE)
+@FieldDefaults(level = AccessLevel.PUBLIC)
+@NoArgsConstructor(force = true, access = AccessLevel.PRIVATE)
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@Builder
+@Embeddable
+public class Coordinates {
```

Comment:
Well done! :1st_place_medal: 
You can further improve with creating our own annotation. Then you don't need to repeat all seven.